cmd/aries-js-worker/README.md

```diff
@@ -27,16 +27,90 @@ Go to [`vue-framework-go`](./vue-aries-framework-go) for a sample agent built wi
 
 View [`index.html`](./index.html) for examples.
 
-**Important:** Make sure the web server adds the appropriate headers when serving the compressed `aries-js-worker.wasm.gz` file:
+### Important - Serving the Assets
+
+`aries-js-worker` loads some assets at runtime: the web assembly binary and a couple of JS scripts. These assets are
+located in the `dist/assets` directory (if you `npm install` it, you'll find them in
+`./node_modules/@hyperledger/aries-framework-go/dist/assets`).
+
+Things that need to work if you are to use `aries-js-worker` on the client side.
+
+#### Headers
+
+Make sure the content server adds the appropriate headers when serving the compressed `aries-js-worker.wasm` file.
+`aries-js-worker` uses the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) to fetch the wasm.
+
+Examples:
+
+**Serving gzipped wasm:**
+
+Headers:
 
 * `Content-Type: application/wasm`
 * `Content-Encoding: gzip`
 
-Consult your web server's documentation for instructions on how to do this (eg. for Apache: [Content-Type](https://httpd.apache.org/docs/2.4/mod/mod_mime.html#addtype), [Content-Encoding](https://httpd.apache.org/docs/2.4/mod/mod_deflate.html#precompressed)).
+**Serving wasm compressed with brotli:**
+
+If your browser supports it, then the headers are:
+
+* `Content-Type: application/wasm`
+* `Content-Encoding: br`
+
+Note, however, that your browser may not support this compression mode.
+ 
+Not all browsers include `br` in `Accept-Encoding` when using `fetch()` (Firefox doesn't) and it is impossible to
+override because `Accept-Encoding` is a [forbidden header name](https://fetch.spec.whatwg.org/#forbidden-header-name).
+
+**Serving uncompressed wasm (not recommended):**
+
+Headers:
+
+* `Content-Type: application/wasm`
+
+#### Path
+
+The URL used to fetch the WASM file is **always** `<assetsPath>/aries-js-worker.wasm`.
+This path needs to exist even if your content server is serving a compressed version.
+
+#### Configuring your content server
+
+Here are some examples:
+
+**Nginx**
+
+[Sending compressed files](https://docs.nginx.com/nginx/admin-guide/web-server/compression/#sending-compressed-files):
+enabling `gzip_static` on a location will automatically serve requests to `http://example.com/assets/aries-js-worker.wasm`
+with `aries-js-worker.wasm.gz` if it exists.
+
+Example: Nginx serving your assets under `/public/assets` with gzipped wasm:
```

Comment:
When I configured nginx, I did the following:

- Enable `gzip_static on;` for the location of `aries-js-worker\.wasm`
- Added application/wasm to the mime.types file of nginx.

(didn't have to the the example you are showing below.)